### DISCUSSION TITLE
Icon guidelines

### DISCUSSION THREAD

**Discussion by naiyerasif** [zed-industries/zed#6942] [Feature Requests]
To make things easier for contributors, it'd be a good idea to lay down some guidelines for new icons. Here's what I think may be good starting point while staying close to the existing icon set.
- `viewBox` should be `0 0 14 14` with **2px padding** around icons
- Icons should be optically centered (rather than geometrically centered)
- The submitted SVG should be optimized through [svgo](https://svgo.dev/) (You can run an SVG through `svgo` online with [SVGOMG](https://jakearchibald.github.io/svgomg/) or [SVG Viewer](https://www.svgviewer.dev/))

I don't think setting any specific expectations for `stroke` / `fill` makes sense (especially when icons reflect an external branding) but feel free to provide anything else that might be helpful.

> I'm not sure if this belongs to Q&A but this seems to be the most appropriate category available at this moment.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
